addons/danger/functions/fnc_leaderAssess.sqf

```diff
@@ -12,20 +12,20 @@ if (_pos isEqualTo []) then {
 };
 
 // settings
-private _mode = toLower (group _unit getVariable [QGVAR(dangerAI),"enabled"]);
+private _mode = toLower (group _unit getVariable [QGVAR(dangerAI), "enabled"]); // joko: @nKenny convert to Boolean?
```

Comment:
I agree with converting this to boolean unless it is used as enum

Reply:
The reason not to convert to Boolean is because this variable can be used to enable other AI modes: Such as 'militia rush', 'recon fall back', or 'insurgent'. While none of these have been created, this is to remain future proof. 

The test isn't run that often.